CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ option(DEQP_ENABLE_GLES3	"Build OpenGL ES 3.0 tests" ON)
 option(DEQP_ENABLE_GLES31	"Build OpenGL ES 3.1 tests" ON)
 option(DEQP_ENABLE_VULKAN	"Build Vulkan tests" ${NOT_EMSCRIPTEN})
 option(DEQP_TEST_EGL		"Build EGL tests" ${NOT_EMSCRIPTEN})
-# TODO: Remove EGL dependency from Emscripten build.
+# TODO(kainino@chromium.org): Remove EGL dependency from Emscripten build.
```

Comment:
Oh now I understood your email message - by this comment I was thinking `"remove mandatory EGL dependency"`, originally thinking that we would want to test both aspects, i.e. make the suite build without EGL where it'd utilize "native WebGL" context creation API that Emscripten provides, but also the EGL dependency. That's because the native WebGL context creation has the extra init parameters it can offer, such as the prefer high power or major performance caveats params.